app/Http/Controllers/ExportController.php

```diff
@@ -2,86 +2,86 @@
 
 namespace App\Http\Controllers;
 
-use App\{Guild, Item};
-use Illuminate\Support\Facades\DB;
-use Illuminate\Support\Facades\Cache;
+use Exception;
+use App\{Expansion, Guild, GuildItem, Item};
+use Illuminate\Support\Facades\{App, DB, Cache};
 
 class ExportController extends Controller {
-    const CSV  = 'csv';
-    const HTML = 'html';
-    const JSON = 'json';
-
-    const ADDON_HEADERS = [
-        "type",
-        "character_name",
-        "character_class",
-        "character_is_alt",
-        "character_inactive_at",
-        "character_note",
-        "sort_order",
-        "item_id",
-        "is_offspec",
-        // "officer_note",
-        "received_at",
-        "item_prio_note",
-        "item_tier_label",
+    public const CSV  = 'csv';
+    public const HTML = 'html';
+    public const JSON = 'json';
+
+    public const ADDON_HEADERS = [
+        'type',
+        'character_name',
+        'character_class',
+        'character_is_alt',
+        'character_inactive_at',
+        'character_note',
+        'sort_order',
+        'item_id',
+        'is_offspec',
+        // 'officer_note',
+        'received_at',
+        'item_prio_note',
+        'item_tier_label',
     ];
 
-    const LOOT_HEADERS = [
-        "type",
-        "raid_group_name",
-        "member_name",
-        "character_name",
-        "character_class",
-        "character_is_alt",
-        "character_inactive_at",
-        "character_note",
-        "sort_order",
-        "item_name",
-        "item_id",
-        "is_offspec",
-        "note",
-        // "officer_note",
-        "received_at",
-        "import_id",
-        "item_note",
-        "item_prio_note",
-        "item_tier",
-        "item_tier_label",
-        "created_at",
-        "updated_at"
+    public const LOOT_HEADERS = [
+        'type',
+        'raid_group_name',
+        'member_name',
+        'character_name',
+        'character_class',
+        'character_is_alt',
+        'character_inactive_at',
+        'character_note',
+        'sort_order',
+        'item_name',
+        'item_id',
+        'is_offspec',
+        'note',
+        // 'officer_note',
+        'received_at',
+        'import_id',
+        'item_note',
+        'item_prio_note',
+        'item_tier',
+        'item_tier_label',
+        'created_at',
+        'updated_at'
     ];
 
-    const ITEM_NOTE_HEADERS = [
-        "name",
-        "id",
-        "instance_name",
-        "source_name",
-        "guild_note",
-        "prio_note",
-        "tier",
-        "tier_label",
-        "created_at",
-        "updated_at",
+    public const ITEM_NOTE_HEADERS = [
+        'name',
+        'id',

```

Comment:
Keeping consistent with the other exports this should probably be `item_id`

Reply:
Yeah agreed, but when refactoring I try to change little to (preferably) nothing when it comes to functionality to reduce risks :) (refactors are usually hard to test already because they aren't contained to isolated pieces of code).

Reply:
I must be missing something. `id` _is_ a new field, and we can name it whatever we want in this case; no?

Reply:
Just double checked on your master branch, but `id` is definitely in there :)

![image](https://user-images.githubusercontent.com/4224975/126079097-3be307eb-9142-41b4-9ef9-e1edcf3f80a9.png)

Reply:
Oh; my mistake. I misread the change on github and thought that it was added rather than just the quotes being changed. Not sure why it highlighted `id` differently from `name`.
![image](https://user-images.githubusercontent.com/3407841/126209326-4d269505-2809-4a0e-aa03-180e494d9833.png)